validator/client/grpc-api/grpc_validator_client.go

```diff
@@ -143,13 +147,80 @@ func (grpcValidatorClient) GetAggregatedSelections(context.Context, []iface.Beac
 }
 
 func NewGrpcValidatorClient(cc grpc.ClientConnInterface) iface.ValidatorClient {
-	return &grpcValidatorClient{ethpb.NewBeaconNodeValidatorClient(cc)}
-}
-
-func (c *grpcValidatorClient) StartEventStream(context.Context) error {
-	panic("function not supported for gRPC client")
+	return &grpcValidatorClient{ethpb.NewBeaconNodeValidatorClient(cc), false}
+}
+
+func (c *grpcValidatorClient) StartEventStream(ctx context.Context, topics []string, eventsChannel chan<- *eventClient.Event) {
+	ctx, span := trace.StartSpan(ctx, "validator.gRPCClient.StartEventStream")
+	defer span.End()
+	if len(topics) == 0 {
+		eventsChannel <- &eventClient.Event{
+			EventType: eventClient.EventError,
+			Data:      []byte(errors.New("no topics were added").Error()),
+		}
+		return
+	}
+	// TODO: ONLY WORKS WITH HEAD TOPIC RIGHT NOW/ONLY PROVIDES THE SLOT
```

Comment:
would work correctly if we had a temporary gRPC implementation

Reply:
I think this is fine :-)